R/example_package.R

```diff
@@ -9,12 +9,24 @@
 #'    `"path": ["observations_1.tsv", "observations_2.tsv"]`.
 #' 3. `media`: inline data stored in `data`.
 #'
+#' @param version Version of the [Data Package](
+#' https://datapackage.org/blog/2024-06-26-v2-release/) standard that this
+#' example Data Package uses. Currently only version 1 is supported.
 #' @return A Data Package object, see [create_package()].
 #' @family sample data
 #' @export
 #' @examples
 #' example_package()
-example_package <- function() {
-  path <- system.file("extdata", "datapackage.json", package = "frictionless")
+example_package <- function(version = 1) {
```

Comment:
Version should be a string value `"1.0"` Use camtrapdp logic to check for version compatibility:

https://github.com/inbo/camtrapdp/blob/573e4b822718ba573f310bf2099e6f9311079e53/R/read_camtrapdp.R#L48-L49